src/main/fc/fc_rc.c

```diff
@@ -205,35 +205,35 @@ void processRcCommand(void)
         }
 
         if (isRXDataNew) {
-            rcInterpolationFactor = rxRefreshRate / targetPidLooptime + 1;
-
-            if (debugMode == DEBUG_RC_INTERPOLATION) {
-                for(int axis = 0; axis < 2; axis++) debug[axis] = rcCommand[axis];
-                debug[3] = rxRefreshRate;
-            }
+            rcInterpolationStepCount = rxRefreshRate / targetPidLooptime;
 
             for (int channel=ROLL; channel < interpolationChannels; channel++) {
-                deltaRC[channel] = rcCommand[channel] -  (lastCommand[channel] - deltaRC[channel] * factor / rcInterpolationFactor);
-                lastCommand[channel] = rcCommand[channel];
+                rcStepSize[channel] = (rcCommand[channel] - rcCommandInterp[channel]) / (float)rcInterpolationStepCount;
```

Comment:
division by zero may happen here. It will be probably harmless, but it's better to catch it